airgun/entities/discoveryrule.py

```diff
@@ -50,13 +50,16 @@ def read(self, entity_name, widget_names=None):
         return view.read(widget_names=widget_names)
 
     def read_all(self):
-        """Reads the whole discovery rules table.
+        """Reads the entire discovery rules table or page after a rule is deleted
 
         :return: list of table rows, each row is dict,
```

Comment:
Can you please update what it returns?

Reply:
Done.

Reply:
Thanks. Also, please update :return: according to new implementation

Reply:
The changes have been made.